CHANGELOG.md

```diff
@@ -1,6 +1,257 @@
 Changelog
 =========
 
+## [43.0.0](https://github.com/ckeditor/ckeditor5/compare/v42.0.2...v43.0.0) (August 7, 2024)
+
+We are happy to announce the release of CKEditor 5 v43.0.0.
+
+### Release highlights
+
+#### Merge fields
+
+The new [merge fields feature](https://ckeditor.com/docs/ckeditor5/latest/features/merge-fields.html) is a game-changer for creating dynamic documents. Imagine being able to insert placeholders in your content, indicating where specific values should go, without showing the actual values. This is perfect for crafting email templates or document forms, which you can later export as Word or PDF files filled with real content.
+
+There are exciting ways you can use merge fields:
+
+1. **Creating Templates:** Easily insert merge fields as placeholders. The default preview mode shows these placeholders, giving you a clear layout without any data distraction.
+2. **Previewing with Example Data:** Want to see how your document will look with actual data? Switch the preview mode to display sample data, giving you a sneak peek of the final result.
+3. **Editing Merged Documents:** Start with a blank document or a pre-made template, and see placeholders automatically fill with real data. This mode shows the actual data instead of placeholders, streamlining the editing process.
+
+Read more in the [merge fields documentation](https://ckeditor.com/docs/ckeditor5/latest/features/merge-fields.html).
+
+#### Export to Word V2 as the default version
+
+The V2 version of the [export to Word feature](https://ckeditor.com/docs/ckeditor5/latest/features/converters/export-word.html) introduced significant improvements, optimizations, and fixes. This update enhances the overall performance, making the export process faster and more reliable, especially for large documents. Key improvements include better handling of table borders, automatic detection of Word styles from CSS, and support for more text-related CSS properties, ensuring your documents maintain their intended formatting.
+
+Starting this version, the V2 configuration becomes the default in the `ExportWord` plugin. Make sure to [migrate your configuration](https://ckeditor.com/docs/ckeditor5/latest/updating/guides/update-to-43.html#export-to-word-v2-becomes-the-default) if you are using it.
+
+#### Improved sidebar accessibility and navigation
+
+We have introduced multiple enhancements to make the sidebar more accessible. You can now use `Shift+Ctrl+E` to move focus from the editor to the active annotation. Navigation has been improved with `Arrow Up` and `Arrow Down` for annotations and comments archive.
```

Comment:
```suggestion
We have introduced multiple enhancements to make the sidebar more accessible. Among other improvements, you can now use `Shift+Ctrl+E` to move focus from the editor marker to the active annotation. When your comment reply is ready, you can quickly submit it using a new `Ctrl+Enter` shortcut. Also, navigation from one annotation to another is now possible by using arrow keys.

The full lists of keyboard supported actions can be reviewed in our [Accessibility support guide](https://ckeditor.com/docs/ckeditor5/latest/features/accessibility.html#Keystrokes-for-interacting-with-annotation-threads-e-g-comments-track-changes-suggestions).
```

Reply:
I am not sure if the link is correct I can verify it later or we can simplify it.